source/write/replace.txt

```diff
@@ -0,0 +1,282 @@
+         For more information, see :manual:`Schema Validation </core/schema-validation/#schema-validation>` in the {+mdb-server+} manual.
+       | Defaults to ``false``.
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``comment``
+     - | Specifies a comment of any valid BSON type to attach to the operation. 
+         Once set, this comment appears alongside records of this command in the following locations:
+
+       - :ref:`mongod log messages <log-messages-ref>`, in the
+         ``attr.command.cursor.comment`` field
+       - :ref:`Database profiler output <profiler>`, in the :data:`system.profile.command <system.profile.command>` ``comment`` field
+       - :dbcommand:`currentOp` output, in the :data:`currentOp.command
+         <currentOp.command>` ``comment`` field
+
+       | For more information, see the :manual:`insert Command
+         Fields guide </reference/command/insert/#command-fields>` in the {+mdb-server+} manual.
+
+   * - ``hint``
+     - | Specifies the index to scan for documents that match the query filter.
+         For more information, see the
+         :manual:`Update Statement fields guide </reference/command/update/#std-label-update-statement-documents>`
+         in the {+mdb-server+} manual. 
+
+   * - ``let``
+     - | Specifies a document containing variables and their values to be used in the ``replace_one()`` method. 
+         This allows you to improve code readability by separating the variables from the operation text. 
+         Values must be constant or closed expressions that don't reference document fields. 
+         For more information, see the 
+         :manual:`update Command Fields guide </reference/command/update/#std-label-replace-let-syntax>`
+         in the {+mdb-server+} manual.
+
+   * - ``upsert``
+     - | Specifies whether the replace operation performs an upsert operation if no 
+         documents match the query filter.
+       | Defaults to ``false``.
+
+   * - ``write_concern``
+     - | Sets the write concern for the operation.
+         For more information, see :manual:`Write Concern </reference/write-concern/>`
+         in the {+mdb-server+} manual.
+
+Example: ``hint``
+^^^^^^^^^^^^^^^^^
+
+The following example uses the :ref:`create_index() <cpp-indexes>` method to create an ascending single-field index 
+on the ``name`` field. It then instantiates a ``mongocxx::options::replace`` object called ``opts`` and sets its ``hints`` field value to the name 
+of the new index, ``name_1``. When ``opts`` is passed in as an optional argument, the ``replace_one()`` method searches the ``name_1`` index for a
+``name`` field value of ``Nobu`` and replaces the associated document with a new document that has a ``name`` field value of ``La Bernadin``.  
```

Comment:
S: I think you can condense this description a little bit. We try to describe what's happening in the code conceptually more than describing the actual code text (names of variables, for example), so you could edit to something like:
```suggestion
The following example uses the ``create_index()`` method to create an ascending single-field index 
on the ``name`` field. It then passes a ``mongocxx::options::replace`` object to the ``replace_one()`` method and sets its ``hint`` field to the new index. This instructs the operation to use the ``name`` field index when replacing the document with a ``name`` value of ``"Nobu"``:
```

Reply:
And since you have the link below to the Indexes guide, I don't think you need the inline link on create_index()